desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -104,20 +104,25 @@ def handle_on_link_shared(channel_id, message_ts, links):
     id_type, qid = urlsplit(item['url'])[3].split('=')
 
     try:
-      if path == '/hue/editor' and id_type == 'editor':
+      if path == '/hue/editor' and id_type == 'editor' and qid.isdigit():
```

Comment:
nit: for the record it starts to be non consistent here (a uuid for an editor can come from a saved query), then can handle both depending if id or uuid like you did below in the  id_type or check_document_access_permission()